client/components/domains/register-domain-step/index.jsx

```diff
@@ -498,6 +501,51 @@ class RegisterDomainStep extends Component {
 		);
 	}
 
+	getActiveIndexByKey( items, tlds ) {
```

Comment:
Although `ResponsiveToolbarGroup` only accepts one active child at a time, we can improve the user experience a bit by connecting both filters when possible (if the user selects search filter `.blog`, we'll select `.blog` at the bottom filter as well).

Reply:
Personally, I thought that only one filter was selectable because I had an intuition that UI like this has an interaction of selecting only one option.

Should we explore more into this? If we decide to do multi-select, we can do it in a separate PR.

Reply:
Agree, we can do a multi-select in a separate PR after reviewing it with the design team.